app/controllers/api/v1/recipes_controller.rb

```diff
@@ -1,36 +1,41 @@
 class Api::V1::RecipesController < ApplicationController
   before_action :authenticate_user!, :except => [:show, :index]
 
-  def show
-    @recipe = Recipe.find_by!(id: params[:id])
-    render json: @recipe, :include => [{comments: {include: :user }}, :tags]
-  end
+    def show
```

Comment:
I assume this was Prettier or some other automated thing, but it looks like you've changed the indentation throughout this file. It was correct before, so if we can revert those changes, that'd be awesome.